app/src/org/commcare/android/tasks/ResourceEngineTask.java

```diff
@@ -273,45 +304,17 @@ protected ResourceEngineOutcomes doTaskBackground(String... profileRefs) {
         } catch (Exception e) {
             e.printStackTrace();
 
-            tryToClearApp();
-
-            Logger.log(AndroidLogger.TYPE_ERROR_WORKFLOW, "Unknown error ocurred during install|" + e.getMessage());
+            Logger.log(AndroidLogger.TYPE_ERROR_WORKFLOW,
+                    "Unknown error ocurred during install|" + e.getMessage());
             return ResourceEngineOutcomes.StatusFailUnknown;
         }
     }
 
-    /**
-     *  For now, never clear automatically - just let user choose when to retry vs. resume
-     */
-    protected void tryToClearApp(){
-        //if(partialMode == false && upgradeMode == false){}
-    }
-
-    /* (non-Javadoc)
-     * @see android.os.AsyncTask#onProgressUpdate(Progress[])
-     */
     @Override
-    protected void onProgressUpdate(int[]... values) {
-        super.onProgressUpdate(values);

```

Comment:
Removed this because it just calls its super. Lost some generics annotations, but otherwise should be fine, right?